.github/workflows/triage-issue.yml

```
@@ -0,0 +1,19 @@
+name: Triage Issue
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+jobs:
+  label_issues:
+    runs-on: windows-latest
```

Could you please change this to `ubuntu-latest`?

```suggestion
    runs-on: ubuntu-latest
```